### DISCUSSION TITLE
How to support translating?

### DISCUSSION THREAD

**Discussion by WeT-Klb** [nocodb/nocodb#1144] [General]
Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **mysql2**
ProjectOnRootDB: **true**
RootDB: **mysql2**
PackageVersion: **0.84.10**

Is it possible to support tranlating NocoDB? The translation into german language is really not good.
"Table" is translated to "Tisch" which means like "Dining table" ;-)

**Comment by o1lab** [comment]
@WeT-Klb : thank you for help point this. 


@dstala has done improvements to process lately (so this will get better) - I would suggest to take a copy of this [spreadsheet](https://docs.google.com/spreadsheets/d/1kGp92yLwhs1l7lwwgeor3oN1dFl7JZWuQOa4WSeZ0TE/edit#gid=2076107172) - make all the changes necessary for German. Make your spreadsheet publicly accessible and share the link with us. And we will take the changes forward.

**Comment by WeT-Klb** [comment]
Hello,

 i tried to translate everything to german language - can you have a look on the spreadsheet?

I added 4 new tabs with the modified translation. Everything I changed I marked in yellow colour.
[noco-i18n-modified german.xlsx](https://github.com/nocodb/nocodb/files/8038836/noco-i18n-modified.german.xlsx)

**Reply by wingkwong** [reply]
Please take a look at [i18n Translations](https://github.com/nocodb/nocodb/issues/633).

**Reply by WeT-Klb** [reply]
First of all the link i was given yesterday by @o1lab was an other Spreadsheet like the recent one.
Now i followed @wingkwong  link, made a copy, created a share link, made my changes, downloaded as CSV-file, went to the upload-page, choose my CSV-file, selected my language - but that´s it - no where to insert the created link.
![Upload](https://user-images.githubusercontent.com/20600342/153382853-4848c2b3-7b76-4a44-853f-b17690f31f7b.jpg)

So if you want to use my translation - i added it here as attachment.
[noco-i18n - [v2.0] i18n.csv](https://github.com/nocodb/nocodb/files/8039906/noco-i18n.-.v2.0.i18n.csv)
[noco-i18n.xlsx](https://github.com/nocodb/nocodb/files/8039949/noco-i18n.xlsx)

The link i was given is:
https://docs.google.com/spreadsheets/d/1ABXxzRgTokWyr_zsvATNBzCwOCqviat2L-MkmrVd214/edit?usp=sharing

**Reply by wingkwong** [reply]
The link given by @o1lab is the master spreadsheet which is same as the one found in my link. We will take a look at it and merge it later. Thanks for the help.

**Reply by dstala** [reply]
@WeT-Klb Thanks for the translations. I will integrate into main line today.

**Reply by WeT-Klb** [reply]
I think in CSV-File only one tab is updated. Please check the xlsx-file for 2nd tab too (with the formula for GOOGLETRANSLATE there).

**Reply by dstala** [reply]
@WeT-Klb Changes committed from i18n spreadsheet. We are in the process of making all corresponding changes in the front end too. So, some messages could still be displayed in English.
You have made good changes in the shared sheet spending considerable energy! Thanks for your efforts!
Continue to support us!

https://github.com/nocodb/nocodb/commit/3a0caf0f850fcfff024adab91e7c12d0daf1f503

**Comment by dstala** [comment]
Changes are reflected now in our master-spreadsheet
https://docs.google.com/spreadsheets/d/1kGp92yLwhs1l7lwwgeor3oN1dFl7JZWuQOa4WSeZ0TE/edit#gid=2076107172

Note: Highlighted in green box/ red font are 'manual corrections', deviating from GOOGLE TRANSLATE

**Comment by WeT-Klb** [comment]
Ooops - i saw one mistake.

Line 466 should be "Ansicht erfolgreich erstell**t**" instead of "Ansicht
erfolgreich erstell**e**".

**Reply by dstala** [reply]
Accepted. Changes submitted. Thanks for review :)

**Reply by WeT-Klb** [reply]
Here is the most recent translation - found few more mistakes - sorry.
[noco-i18n.xlsx](https://github.com/nocodb/nocodb/files/8041517/noco-i18n.xlsx)

**Reply by dstala** [reply]
@WeT-Klb Please check if the corrections are as identified below
https://docs.google.com/spreadsheets/d/1bsmWrbNnAlNeimWalxK1LwEnahIGQQd0kJ_OUGP-Cdk/edit?usp=sharing

**Reply by WeT-Klb** [reply]
Yes - exactly.

**Reply by dstala** [reply]
done! https://github.com/nocodb/nocodb/commit/902a3afc76a4ef66a4bc0c7b3670db5c75f5bc66
thanks again!

**Reply by WeT-Klb** [reply]
Whenever i can help with german translation - just let me know. I´m glad to help. :-)

**Reply by o1lab** [reply]
Awesome, thanks a tonne.

There was another [PR](https://github.com/nocodb/nocodb/pull/1149/files) for German.

If you could confirm you'd it covered or you want to merge provide this changes via spreadsheet - that would be great.

**Reply by WeT-Klb** [reply]
I checked every single line for german translation in the Excel-File i sent to you. I would say if you don´t mind just take my sheet and use it 1:1 - and everything should be done. I don´t want to sound selfish - but that other translation somehow just don`t feel right.
Is there a way maybe i can help with the german ReadMe-File, too?

**Reply by o1lab** [reply]
Thank you so much - really appreciate it. 

Please send PR for German readme as well.

**Comment by WeT-Klb** [comment]
Is there like a spreadsheet too or do i just need to download the "readme.md" and work on it?

**Reply by o1lab** [reply]
The readme.md can be just worked up on.

If you think there is a need for change in banner images text - please change the text in [spreadsheet](https://docs.google.com/spreadsheets/d/1kGp92yLwhs1l7lwwgeor3oN1dFl7JZWuQOa4WSeZ0TE/edit#gid=122217622). And for submitting spreadsheet pls see the procedure [here](https://docs-dev.nocodb.com/engineering/translation).

**Reply by WeT-Klb** [reply]
Can you check the changes in tab v1.0 banner-images? Last time you only show the changes in i18n, but i´m not sure you had a look to "v1.0 banner-images" too.
[noco-i18n.xlsx](https://github.com/nocodb/nocodb/files/8055539/noco-i18n.xlsx)

The german Readme.MD i did as suggested.

**Reply by o1lab** [reply]
Ah cool, @dstala will take care of changes in banner images from your excel. I guess that part was missed.

We look forward to your German readme changes.

**Comment by WeT-Klb** [comment]
If there are any questions - just let me know. J

**Reply by WeT-Klb** [reply]
Is there something wrong with my german "Read.Me"-translation as there still is the old one?
When i did it i even could see the changes i did online - now i can´t see it anywhere anymore.

**Reply by WeT-Klb** [reply]
@o1lab 
Is something wrong with my translations? I can´t see them being merged into last docker release nor can i see changes in german read.me.

**Reply by o1lab** [reply]
Could you do a hard refresh on your browser once ? cmd+shift+r or ctrl+shift+r

**Reply by wingkwong** [reply]
@WeT-Klb your changes are here. Home -> Start.

![image](https://user-images.githubusercontent.com/35857179/154976812-696be63b-5cef-44c7-a762-53a4cb1fcbb9.png)

**Reply by WeT-Klb** [reply]
@o1lab 
 https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/languages/german.md the translation still wasn´t taken over, It´s very easy to see as under "Deploy to Heroku"-button everything is still in english language and further still "Build Automationen mit unserem App Store" instead of "Baue Automationen...".

@wingkwong 
i can see the changes - but i feel they didn´t find the way into the docker file 0.84.12.

**Comment by dstala** [comment]
@WeT-Klb 
Apologies. Read me changes, I hadn't merged them yet. I will do that tomorrow.
For the rest of i18n translations - we are in the process of updating the corresponding code in the front end. Hence, part of it would still be in English.

**Reply by WeT-Klb** [reply]
I see . As i said before - if you need my help for translation something - just let me know. I´m glad if i can support you.

**Comment by dstala** [comment]
@WeT-Klb Could you help me identify your latest spreadsheet consisting of banner image changes? I did not find any changes here
https://github.com/nocodb/nocodb/files/8055539/noco-i18n.xlsx
https://github.com/nocodb/nocodb/files/8041517/noco-i18n.xlsx

**Comment by WeT-Klb** [comment]
In fact both are right - i only adapted the words in formulas - but should be the latest one.

 

=WENNFEHLER(__xludf.DUMMYFUNCTION("IF(OR(ISBLANK($B8),ISBLANK(F$1)),"""",GOOGLETRANSLATE($B8,$C$1, F$1))");"Automationen bauen mit unserem App Store.")

 

Only in 

 

=WENNFEHLER(__xludf.DUMMYFUNCTION("IF(OR(ISBLANK($B10),ISBLANK(F$1)),"""",GOOGLETRANSLATE($B10,$C$1, F$1))");"3RD-Party-Apps-Integrationen mit API-Token bauen") should be "3rd-Party…..)

 

If it is more easy I can send a whole new file again with all the done tranlations.

 

 

Von: Raju Udava ***@***.*** 
Gesendet: Dienstag, 22. Februar 2022 12:20
An: nocodb/nocodb
Cc: Werner; Mention
Betreff: Re: [nocodb/nocodb] How to support translating? (Discussion #1144)

 

@WeT-Klb <https://github.com/WeT-Klb>  Could you help me identify your latest spreadsheet consisting of banner image changes? I did not find any changes here
https://github.com/nocodb/nocodb/files/8055539/noco-i18n.xlsx
https://github.com/nocodb/nocodb/files/8041517/noco-i18n.xlsx

—
Reply to this email directly, view it on GitHub <https://github.com/nocodb/nocodb/discussions/1144#discussioncomment-2227877> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AE5FMFQ5IMNFFSAEBYOSW3LU4NWPVANCNFSM5N6ISG2Q> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Reply by dstala** [reply]
https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/languages/german.md

Incorporated

**Comment by WeT-Klb** [comment]
I tried to tell where i did the translation - but not only those examples.

Please take a look on all  german fields in that spreadsheet.

The yellow marks are only to show where I changed something within that translate-formula.

Sorry for bad explaination

 

 

Von: Raju Udava ***@***.*** 
Gesendet: Dienstag, 22. Februar 2022 15:47
An: nocodb/nocodb
Cc: Werner; Mention
Betreff: Re: [nocodb/nocodb] How to support translating? (Discussion #1144)

 

https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/languages/german.md

Incorporated

—
Reply to this email directly, view it on GitHub <https://github.com/nocodb/nocodb/discussions/1144#discussioncomment-2229129> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AE5FMFWZN7XHYE6THYJLBRLU4OOXTANCNFSM5N6ISG2Q> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Reply by dstala** [reply]
Somehow don't see any yellow marks on the spreadsheet & the formula remains consistent with earlier version. No change. The contents that I see are dumped below (for en & de language codes).
Help me correct them inline here or via spreadsheet. 

<google-sheets-html-origin><style type="text/css"></style>

en | de
-- | --
Rich Spreadsheet on any database. | Rich-Tabellenkalkulation in jeder Datenbank.
Gallery View: Makes things visual | Galerieansicht: macht die Dinge visuell
Kanban view: Track progress like a professional | Kanban-Ansicht: Verfolgen Sie den Fortschritt wie ein Profi
Calendar view : Never miss an event | Kalenderansicht: Verpassen Sie niemals ein Ereignis
Collaborate : Just like spreadsheets | Zusammenarbeit: Genau wie Tabellenkalkulationen
Share Views with password protection | Ansichten mit Passwortschutz teilen
Build automations with our app store. | Build Automationen mit unserem App Store.
Programmatic access with APIs | Programmatischer Zugriff mit APIs
Build 3rd party apps integrations with API tokens | 3RD-Party-Apps-Integrationen mit API-Token bauen
Customize pages (coming soon) | Passen Sie Seiten an (kommt in Kürze)
Mobile Apps (coming soon) | Mobile Apps (in Kürze)

**Comment by WeT-Klb** [comment]
Reichhaltige Tabellenkalkulation in jeder Datenbank

Galerieansicht: Macht Dinge visuell

Kanban-Ansicht: Den Fortschritt verfolgen wie ein Profi

Kalenderansicht: Nie ein Ereignis verpassen

Zusammenarbeit: Genau wie mit Tabellenkalkulationen

Ansichten mit Passwortschutz teilen

Automationen bauen mit unserem App-Store

Programmatischer Zugriff mit APIs

3rd-Party-Apps-Integrationen mit API-Tokens bauen

Seiten anpassen (in Kürze)

Mobile Apps (in Kürze)



**_Question:_**
What is the meaning of **Rich** Spreadsheet on any database ?

1) A fix word like **Rich-Text or Excel or Word** or does it mean

2) **rich on possibilities or variations = verry flexible**?

For 1) the tranlation of first item would be: **Rich-Tabellenkalkulation** in jeder Datenbank
for 2) the tranlation of first item would be: **Reichhaltige Tabellenkalkulation** in jeder Datenbank  (this is what translated in the moment in first line)

**Comment by dstala** [comment]
@WeT-Klb Thanks again!
https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/languages/german.md

Changes recorded in noco-i18n master spreadsheet (banner images) as well

**Comment by WeT-Klb** [comment]
Great - now the headlines are there.

But the translation for the words in Read.Me are still missing.

You still can see under 1-Click-Deployment Heroku the words in english language in the german Read.Me:

 

* NocoDB needs a database as input.

* If this input is absent….

 

I did the whole Read.Me inline some days ago  and made the corrections - but unfortunately it seems they dissapeared.

 

Von: Raju Udava ***@***.*** 
Gesendet: Donnerstag, 24. Februar 2022 12:36
An: nocodb/nocodb
Cc: Werner; Mention
Betreff: Re: [nocodb/nocodb] How to support translating? (Discussion #1144)

 

@WeT-Klb <https://github.com/WeT-Klb>  Thanks again!
https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/languages/german.md

—
Reply to this email directly, view it on GitHub <https://github.com/nocodb/nocodb/discussions/1144#discussioncomment-2243378> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AE5FMFWUZWYJU7PBOBALCXDU4YJ27ANCNFSM5N6ISG2Q> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Reply by dstala** [reply]
For inline text changes, request you to raise a PR on this file.
https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/languages/german.md

**Comment by WeT-Klb** [comment]
Hi,

 

that´s exactly the point. I did it there one or 2 weeks ago - but it never
made any changes.

That´s why I am so curios what was going wrong.

 

 

Von: Raju Udava ***@***.*** 
Gesendet: Freitag, 25. Februar 2022 04:39
An: nocodb/nocodb
Cc: Werner; Mention
Betreff: Re: [nocodb/nocodb] How to support translating? (Discussion #1144)

 

For inline text changes, request you to raise a PR on this file.
https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/langua
ges/german.md


Reply to this email directly, view it on GitHub
<https://github.com/nocodb/nocodb/discussions/1144#discussioncomment-2248810
> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AE5FMFSQER7NJECFCHMI2TLU4
32UBANCNFSM5N6ISG2Q> .
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&m
t=8&pt=524675>  or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=u
tm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>
. 
You are receiving this because you were mentioned.
<https://github.com/notifications/beacon/AE5FMFQW5RO6EZDV6ZAJYGLU432UBA5CNFS
M5N6ISG22YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZG
OAARFA2Q.gif> Message ID:
***@***.***>

**Comment by dstala** [comment]
Did you raise a PR? I don't see any PR on Readme changes.
You can share changes here, done on top of latest readme.md file. I will commit on your behalf.

**Comment by dstala** [comment]
@WeT-Klb - New items added to master spreadshet. Highlighted in Yellow. Could you help us verify for German translation?
https://docs.google.com/spreadsheets/d/1kGp92yLwhs1l7lwwgeor3oN1dFl7JZWuQOa4WSeZ0TE/edit#gid=2076107172

**Comment by dstala** [comment]
You can make a copy of master spreadsheet, make changes & submit a PR for corrections (with modified CSV file or Shared spreadsheet link).

**Reply by WeT-Klb** [reply]
To be honest - i don´t know what is a PR.
Here i send the new tranlation fpr german language (you asked me before)
Please be aware I updated both tabs inside that Excel-File.
[noco-i18n-25022022.xlsx](https://github.com/nocodb/nocodb/files/8141048/noco-i18n-25022022.xlsx)

Can you explain the precedure how i can update the german Read.me?

**Reply by dstala** [reply]
Sharing a spreadsheet link or as an attachment here is fine for i18n. I will apply modifications.

For German readme.md file updates, 
- Sign-in to your Github account
- Open [file](https://github.com/nocodb/nocodb/blob/develop/scripts/markdown/readme/languages/german.md) in edit mode 
- Modify text (you can preview modifications as well): note that this file is a markdown file & associated syntax can be found [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
- Once done, click `propose` changes at end of the file.

**Reply by dstala** [reply]
Changes applied

<google-sheets-html-origin><style type="text/css"></style>

de (old) | de (new)
-- | --
Übermitteln | Übertragen
Einfügung | Einfügen
Zeigen | Anzeigen
Alles zeigen | Alles anzeigen
Optionen anzeigen | Optionen einblenden
Menü anzeigen | Menü einblenden
Fall | Ereignis
Zustand | Bedingung
URL. | URL
Null einstellen | Auf Null setzen
Standard setzen | Auf Standard setzen
Mitarbeitersicht | Mitarbeiteransicht
Gesperrter Blick | Gesperrte Ansicht
Appstore | App-Store
Team & Auth. | Team & Auth
Rollen & Benutzermanagement | Rollen- & Benutzermanagement
UI-Zugangskontrolle. | UI-Zugangskontrolle
Benachrichtigen Sie die via | Benachrichtigen mit
Name anzeigen | Namen anzeigen
Sync-Stand | Sync-Status
Aggregat-Funktion. | Globale Funktion
Buchen Sie eine kostenlose Demo | Eine kostenlose Demo buchen
Join /r/NocoDB | /r/NocoDB beitreten
Dokumentverweis | Dokumentenverweis
Wählen Sie die Benutzerrolle aus | Benutzerrolle auswählen
Kindertisch | Child-Tabelle
Kindersäule | Child-Spalte
Update. | Update
Auf löschen | Löschen
OK & Project speichern | OK & Projekt speichern
Tabelle löschen. | Tabelle löschen
Neue Spalte hinzufügen | Neue Zeile hinzufügen
Spalte speichern | Zeile speichern
Neue Reihe einfügen | Neue Zeile einfügen
Geteilte Ansichtsliste | Geteilte Ansichtenliste
Raster-Ansicht erstellen | Gitter-Ansicht erstellen
Eingebettete HTML-Code kopieren | Eingebetteten HTML-Code kopieren
Sponsor uns. | Sponsor uns
Alle Metadaten von den Metatabellen in Meta-Verzeichnis exportieren. | Alle Metadaten von Metatabellen in Meta-Verzeichnis exportieren.
Suche {search} Spalte | Spalten {search} suchen
Suche Modelle | Modelle suchen
Reihen pro Seite | Zeilen pro Seite
Blatt (s) stehen für den Import zur Verfügung | Blatt/Blätter stehen für den Import zur Verfügung
Beginnen mit dem Erstellen eines neuen Projekts | Mit dem Erstellen eines neuen Projektes beginnen
Jeder im Internet mit diesem Link kann sehen | Für Jeden im Internet mit diesem Link sichtbar
Sieht aus, als hätten Sie den Mailer noch nicht konfiguriert! \ n Bitte kopieren Sie den obigen Einladungs-Link und senden Sie ihn an | Es sieht so aus, als hätten Sie den Mailer noch nicht konfiguriert! \ n Bitte kopieren Sie den obigen Einladungs-Link und senden Sie ihn an
Fügen Sie einen Hilfs-Text hinzu | Einen Hilfs-Text hinzufügen
Diese Ansicht wird durch einen privaten Link geteilt | Diese Ansicht wird durch einen persönlichen Link geteilt
Personen mit einem privatem Link können nur Zellen sehen, die in dieser Ansicht angezeigt werden | Personen mit einem persönlichen Link können nur Zellen sehen, die in dieser Ansicht angezeigt werden
E-Mail mich unter | E-Mail an mich unter
Mitarbeitern mit Bearbeitungsberechtigung oder höher können die Ansichtskonfiguration ändern | Mitarbeitern mit Bearbeitungsberechtigung oder höher können die Ansichtenkonfiguration ändern
Niemand kann die Ansichtskonfiguration bearbeiten, bis sie freigeschaltet ist. | Niemand kann die Ansichtenkonfiguration bearbeiten, bis sie freigeschaltet ist.
Nur Sie können die Ansichtskonfiguration bearbeiten. Weitere Mitarbeiter-Ansichten sind standardmäßig ausgeblendet. | Nur Sie können die Ansichtenkonfiguration bearbeiten. Weitere Mitarbeiter-Ansichten sind standardmäßig ausgeblendet.
Kann die Datensätze anzeigen und kommentieren, aber nichts bearbeiten | Kann die Datensätze anzeigen und kommentieren, aber nicht bearbeiten
Kann die Einträge anzeigen, aber nichts bearbeiten | Kann die Einträge anzeigen, aber nicht bearbeiten
Schema-Migrationen werden automatisch erstellt. Tabelle erstellen und diese Seite aktualisieren. | Schemen-Migrationen werden automatisch erstellt. Tabelle erstellen und diese Seite aktualisieren.
Wir sind ein winziges Team, das Vollzeit arbeitet, um die NOCODB-Open-Source zu machen. Wir glauben, daß ein Werkzeug wie NOCODB für jeden Problemlöser im Internet frei verfügbar sein sollte. | Wir sind ein winziges Team, welches Vollzeit arbeitet, um NocoDB Open-Source zu machen. Wir glauben, daß ein Werkzeug wie NocoDB für jeden Problemlöser im Internet frei verfügbar sein sollte.
Du wirst der "Super Admin" sein | Sie werden der "Super Admin" sein
Tische Metadaten sind synchron | Tabellen-Metadaten sind synchron
Sie können mehrere COMMA (,) getrennte E-Mails hinzufügen | Sie können mehrere kommagetrennte (,) E-Mails hinzufügen
Geben Sie den Namen ein | Geben Sie den Tabellennamen ein
Standardspalten hinzufügen. | Standardspalten hinzufügen
Den Benutzer gibt es nicht oder er hat keine ausreichenden Rechte, das Schema zu erstellen. | Den Benutzer gibt es nicht oder er hat keine ausreichenden Rechte, um das Schema zu erstellen.
Projekt wurde erfolgreich angehalten | Projekt wurde erfolgreich gestoppt
Projekt begann erfolgreich | Projekt erfolgreich gestartet
Bitte aktivieren Sie das SMTP-Plugin im App Store, um die E-Mail-Benachrichtigung zu aktivieren | Bitte aktivieren Sie das SMTP-Plugin im App-Store, um die E-Mail-Benachrichtigung zu aktivieren

**Reply by dstala** [reply]
https://github.com/nocodb/nocodb/commit/06caf0f034d6661ebcfdc5e8ae2b56f9cd0208b6

**Comment by WeT-Klb** [comment]
Did you see my translation file? I already uploaded on friday last week.

Please be aware I updated both tabs in the spreadsheet.

 

 

 

Von: Raju Udava ***@***.*** 
Gesendet: Freitag, 25. Februar 2022 10:26
An: nocodb/nocodb
Cc: Werner; Mention
Betreff: Re: [nocodb/nocodb] How to support translating? (Discussion #1144)

 

@WeT-Klb <https://github.com/WeT-Klb>  - New items added to master spreadshet. Highlighted in Yellow. Could you help us verify for German translation?
https://docs.google.com/spreadsheets/d/1kGp92yLwhs1l7lwwgeor3oN1dFl7JZWuQOa4WSeZ0TE/edit#gid=2076107172

—
Reply to this email directly, view it on GitHub <https://github.com/nocodb/nocodb/discussions/1144#discussioncomment-2250226> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AE5FMFWHSN6M6NAT34HLV6TU45DMFANCNFSM5N6ISG2Q> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .

**Reply by dstala** [reply]
Will work on it today

**Comment by WeT-Klb** [comment]
@dstala 
Today i translated the german READ.ME again and created a Pull Request #1397.

Can you have a look on it?

**Reply by dstala** [reply]
merged

**Reply by dstala** [reply]
thanks again for your proactive help!
